vm/move-package-manager/src/releasement.rs

```diff
@@ -99,7 +99,7 @@ pub fn handle_release(
     };
 
     let p = Package::new(ms, init_script)?;
-    let package_bytes = bcs_ext::to_bytes(&p)?;
+    let package = bcs_ext::to_bytes(&p)?;
```

Comment:
I don't think so.

In the context of the code here alone, there is no logic or long block of code that would cause it to be mistaken for an `Package` struct.

Second, the type inference information is enough to tell us that this is a `Vec<u8>`.

Reply:
I think #L101 naming as package better than #L102

Reply:
I think `_bytes` is superfluous

Reply:
you can use other naming that variable. some far_fetched place, jolestar will help me review.

Reply:
OK, I'll change it to `blob`.